src/test/java/uk/gov/companieshouse/disqualifiedofficers/search/transformer/DisqualificationItemTransformerTest.java

```diff
@@ -79,7 +81,7 @@ public void corporateItemIsTransformer() {
 
     private Disqualification getDisqualification() {
         Disqualification disq = new Disqualification();
-        disq.setAddress(ADDRESS);
+        disq.setAddress(new Address());
```

Comment:
Probably don't need to instantiate a new address here if we're instantiating it already on line 34? or vice versa